modules/cco-ccoctl-upgrading.adoc

```diff
@@ -113,63 +77,22 @@ endif::google-cloud-platform[]
 $ oc create namespace <component_namespace>
 ----
 
-. Use the `ccoctl` tool to process all `CredentialsRequest` objects in the `credrequests` directory by running the following command:
-+
-ifdef::aws-sts[]
-[source,terminal]
-----
-$ ccoctl aws create-iam-roles \
---name=<name> \
---region=<aws_region> \
---credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests \
---identity-provider-arn=arn:aws:iam::<aws_account_id>:oidc-provider/<cluster_name>-oidc.s3.<aws_region>.amazonaws.com
-----
-+
-where:
-+
---
-** `<name>` is the name used to tag any cloud resources that are created for tracking. For upgrades, use the same value that was used for the initial installation.
-** `<aws_account_id>` is the AWS account ID.
-** `<aws_region>` is the AWS region in which cloud resources will be created.
-** `<aws_account_id>`, `<cluster_name>`, and `<aws_region>` are standard elements of the Amazon Resource Name (ARN) for your cluster, provided here to illustrate the format of an ARN. You can obtain the ARN for your cluster's identity provider from the *Identity Providers* menu in the link:https://console.aws.amazon.com/iam/[AWS IAM console]. 
---
-+
-[NOTE]
-====
-For AWS environments that use alternative IAM API endpoints, such as GovCloud, you must also specify your region with the `--region` parameter.
-
-If your cluster uses Technology Preview features that are enabled by the `TechPreviewNoUpgrade` feature set, you must include the `--enable-tech-preview` parameter.
-====
-endif::aws-sts[]
-ifdef::google-cloud-platform[]
-[source,terminal]
-----
-$ ccoctl gcp create-service-accounts \
---credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests \
---name=<name> \
---project=<gcp_project_id> \
---workload-identity-pool=<name> \
---workload-identity-provider=<name>
-----
-+
-where:
+. Use the `ccoctl` tool to process all `CredentialsRequest` objects in the `credrequests` directory by running the command for your cloud provider. The following commands process `CredentialsRequest` objects:
 +
 --
-** `<path_to_directory_with_list_of_credentials_requests>/credrequests` is the directory containing the files of `CredentialsRequest` manifests to create GCP service accounts.
-** `<name>` is the user-defined name for all created GCP resources used for tracking.
-** `<gcp_project_id>` is the GCP project ID in which cloud resources will be created.
+* {alibaba}: `ccoctl alibabacloud create-ram-users`
+* Amazon Web Services (AWS): `ccoctl aws create-iam-roles`
+* Google Cloud Platform (GCP): `ccoctl gcp create-all`
+* IBM Cloud: `ccoctl ibmcloud create-service-id`
+* Nutanix: `ccoctl nutanix create-shared-secrets`
```

Comment:
Hmmm I like how you are thinking. Ideally, I would actually like to get the full commands in here instead of the minor cop-out of "Refer to the `ccoctl` utility instructions in the installation content for your cloud provider for important platform-specific details about the required arguments and special considerations." It would be more helpful to the user to have all this info handy, as long as I can find a way to present it that is not a giant mess (i.e., the GCP user won't be irritated sifting through all the non GCP stuff).